soc/nxp/mcx/mcxn/Kconfig.defconfig

```diff
@@ -15,6 +15,11 @@ config ROM_START_OFFSET
 config ZTEST_NO_YIELD
 	default y if (PM && ZTEST)
 
+# The existing SAI diver cannot initialize the PLL on the board,
+# so the PLL settings will not be performed in the driver.
+config I2S_HAS_PLL_SETTING
+	default n
```

Comment:
```suggestion
```
Not needed, Kconfigs always default to n

Reply:
fixed

Reply:
Sorry @mcuxted ,
I was wrong here.  Can you please revert this back?  And then remove from boards/nxp/frdm_mcxn947/frdm_mcxn947_mcxn947_cpu0_defconfig again.
```
config I2S_HAS_PLL_SETTING
	default n
```